thefuck/rules/git_stash.py

```diff
@@ -0,0 +1,7 @@
+def match(command, settings):
+    return ('git' in command.script
+            and 'Please commit or stash them.' in command.stderr)
+
+
+def get_new_command(command, settings):
+    return 'git stash && ' + command.script
```

Comment:
Shouldn't it unstash the changes after running the command? Like `git stash pop`?